controllers/apply/awards-for-all/fields.js

```diff
@@ -2522,14 +2725,21 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             name: 'termsPersonPosition',
             label: localise({ en: 'Position in organisation', cy: '' }),
             type: 'text',
-            schema: Joi.string().required(),
+            schema: Joi.string().max(FREE_TEXT_MAXLENGTH.large).required(),
             messages: [
                 {
                     type: 'base',
                     message: localise({
                         en: `Enter the position of the person completing this form`,
                         cy: ''
                     })
+                },
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `Position of the person must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
`Position in organisation must be ${FREE_TEXT_MAXLENGTH.large} characters or less`